operators/src/processing/raster_vector_join/aggregated.rs

```diff
@@ -70,9 +70,14 @@ where
 
         let collection = covered_pixels.collection_ref();
 
+        let covered_bounds = covered_pixels
+            .covered_spatial_bounds()
+            .and_then(|b| b.intersection(&query.spatial_bounds))
+            .unwrap_or(query.spatial_bounds); // TODO: if the intersection is empty, we should skip the query
```

Comment:
there are two cases where you unwrap a None:
1. the collection has no features
2. the query does not intersect the covered pixels

in the first case we could skip, but we don't need the intersection for this.
the second case would be a bug or not?

Reply:
i removed that